sdk/storage/Azure.Storage.DataMovement/src/LocalTransferCheckpointer.cs

```diff
@@ -191,7 +191,7 @@ public override Task<bool> TryRemoveStoredTransferAsync(string id, CancellationT
         /// <summary>
         /// Lists all the transfers contained in the checkpointer.
         /// </summary>
-        /// <param name="cancellationToken"></param>
+        /// <param name="cancellationToken">The cancellation token of the current transfer.</param>
```

Comment:
Is it accurate to describe this as cancellation token of the current transfer? Or is it really the cancellation token of the checkpointer? Will change pending feedback

Reply:
For everywhere that has cancellation token let's use
```
<param name="cancellationToken">
        /// Optional <see cref="CancellationToken"/> to propagate
        /// notifications that the operation should be cancelled.
        /// </param>
        ````